src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -647,12 +650,27 @@ private ImmutableArray<ParameterSymbol> ComputeParameters(DiagnosticBag diagnost
                     diagnostics.Add(ErrorCode.ERR_ParameterIsStaticClass, this.locations[0], propertyType.Type);
                 }
 
-                parameters.Add(new SynthesizedAccessorValueParameterSymbol(this, propertyType, parameters.Count));
+                parameters.Add(new SynthesizedAccessorValueParameterSymbol(this, propertyType, parameters.Count, _property.SetterValueFlowAnalysisAnnotations));
```

Comment:
>_property.SetterValueFlowAnalysisAnnotations [](start = 113, length = 44)

Creation of symbols should do as less binding work as possible. It looks like this code will trigger binding of all attributes applied to the property, we can easily delay that work until the information is really needed..